charts/observation-service/templates/fluentd-hpa.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 spec:
   scaleTargetRef:
     apiVersion: apps/v1
-    kind: Deployment
+    kind: StatefulSet
```

Comment:
@terryyylim, to keep the context - is the eventual plan for the `observation-service` chart to depend on `timber-fluentd` and not define its own fluentd templates?

Reply:
`timber-observation-service` will depend on `timber-fluentd` and `observation-service` chart will be removed entirely.